app/controllers/admin_controller.rb

```diff
@@ -0,0 +1,21 @@
+class AdminController < ApplicationController
+  def index
+    @shelters = Shelter.alphabetical_sort
+    @shelters_pending = Shelter.pending_applications
+  end
+
+  def show
+    @application = Application.find(params[:id])
+  end
+
+  def update
+    relationship = ApplicationPet.where("pet_id = ? AND application_id = ?", params[:pet_id], params[:application_id])
```

Comment:
Since we can be reasonably sure there should be only one ApplicationPet with this given id combo, I'd generally stick with `find_by` instead of `where` because `where` returns an array of records rather than a single record